easybuild/tools/systemtools.py

```diff
@@ -612,6 +611,11 @@ def check_os_dependency(dep):
         RPM: '-q',
     }
     os_name = get_os_name()
+
+    rh_based_os = any(os_name.startswith(x) for x in ['centos', 'fedora', 'redhat', 'rhel', 'sl'])
```

Comment:
Isn't the `startswith('sl')` a bit risky? That's not exactly strict... Slackware for example?

Reply:
It probably is, I just took that from some other part of EasyBuild.
Like tensorflow.py

Reply:
@boegel Better now?